services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/_constants.py

```diff
@@ -1,3 +1,4 @@
 from typing import Final
 
 CONTAINER_NAME: Final[str] = "container"
+MULTIPLE_GIGA: Final[float] = 1e9
```

Comment:
MINOR: why ``MULTIPLE_``? `` 3 * MULTIPLE_GIGA``  reads funny

Reply:
OK, I was a bit debaing about it. Let's drop it.